asserts/device_asserts.go

```diff
@@ -166,8 +175,27 @@ func assembleModel(assert assertionBase) (Assertion, error) {
 		return nil, err
 	}
 
-	for _, mandatory := range modelMandatory {
-		if _, err := checkNotEmptyString(assert.headers, mandatory); err != nil {
+	classic, err := checkOptionalBool(assert.headers, "classic")
+	if err != nil {
+		return nil, err
+	}
+
+	if classic {
+		_, ok := assert.headers["kernel"]
+		if ok {
```

Comment:
(nitpick^2) this could be written in a single line: `if _, ok := assert.headers["kernel"]; ok {...`.